editor/js/libs/app/index.html

```diff
@@ -16,13 +16,16 @@
 		</style>
 	</head>
 	<body ontouchstart="">
+		<script src="../examples/js/vr/HelioWebXRPolyfill.js"></script>
```

Comment:
Published project will not find `HelioWebXRPolyfill` under this path.

Reply:
Nice catch! I will update it. Should be `./HelioWebXRPolyfill.js`

Reply:
Fixed